src/PuppeteerRunnerExtension.ts

```diff
@@ -106,6 +107,28 @@ export class PuppeteerRunnerExtension extends RunnerExtension {
     };
 
     switch (step.type) {
+      case 'doubleClick':
```

Comment:
could we write the following?

```
case 'doubleClick':
case 'click':
// clickCount: step.type === doubleClick ? 2 : 1
```

I think apart from this the code is identical.

Reply:
`doubleClicks` do not have a duration (it won't double click if the duration is too long).

Reply:
could we add `delay: step.type === doubleClick ? step.duration : undefined`?